lib/jsonite/lets_proxy.rb

```diff
@@ -0,0 +1,23 @@
+class Jsonite
+  class LetsProxy < BasicObject
+
+    undef_method :==, :!, :!=
+
+    def initialize object, context, lets = {}
+      @__object, @__context, @__lets = object, context, lets
```

Comment:
We use `_`-prepended variables above, and `__`- here. Should we just pick one and be consistent?